docs/03-dev-howtos/20-update-overlay-images.md

```diff
@@ -0,0 +1,19 @@
+Updating Social Media Overlay Images
+====================================
+
+We use the [overlay capability of Fastly IO](https://docs.fastly.com/api/imageopto/overlay) to generate images for social
+media - [here]() is an example with the guardian logo overlayed on it. The code that generates the urls for these images
```

Comment:
minor: looks like the url is missing inside the link markdown?

major: dev howtos! Thank you thank you thank you for writing docs 👏 👏 👏

Reply:
Nice spot, thanks Kate!